### PR TITLE
Convert KATS tests to JSON

### PR DESCRIPTION
**Pull request by chris-wood**
This will let us easily extend the set of test vectors later on.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -49,4 +51,4 @@ harness = false
 [patch.crates-io]
 blind-rsa-signatures = { git = "https://github.com/jedisct1/rust-blind-rsa-signatures", branch="master" }
 voprf = { git = "https://github.com/facebook/voprf",  branch = "main"  }
-p384 = { git = "https://github.com/RustCrypto/elliptic-curves", rev = "136fed7944d53c0508b1a93cd97bdab46891bcf7" }
+p384 = { git = "https://github.com/RustCrypto/elliptic-curves", rev = "136fed7944d53c0508b1a93cd97bdab46891bcf7" }
```
Comment (raphaelrobert) [inline Cargo.toml:54]: Readd the newline?

File: Cargo.toml [hunk 2]
```diff
@@ -17,11 +17,13 @@ async-trait = "0.1.56"
 base64 = "0.21.0"
 generic-array = "0.14.5"
 rand = "0.8.5"
+serde_json = "1.0"
 serde = "1.0.138"
 sha2 = "0.10.2"
 thiserror = "1.0.31"
 tls_codec = "0.2.0"
 voprf = "0.5.0-pre.2"
+hex = { version = "0.4.3", features = ["serde"]}
```
[the remark this answers is not in the document]
Comment (chris-wood) [reply Cargo.toml]: Done 👍

File: Cargo.toml (hunk 1 above)
Comment (chris-wood) [reply Cargo.toml]: Done 👍

**Comment by chris-wood** [comment]
See the latest test vectors in [this PR](https://github.com/cloudflare/pat-go/pull/31).

**Review by raphaelrobert — APPROVED** [review]

**Pull request merged**